controllers/courses.js

```diff
@@ -717,8 +717,9 @@ router.get('/:courseId/', async (req, res, next) => {
 					: course.name,
 				activeTab: req.query.activeTab,
 				lessons,
-				homeworks: homeworks.filter(task => !task.private),
-				myhomeworks: homeworks.filter(task => task.private),
+				assignedHomeworks: homeworks.filter(task => !task.private && !task.submissions),
+				homeworksWithSubmission: homeworks.filter(task => !task.private && task.submissions),
```

Comment:
Maybe you could make the filters dependent on the role of the user. (Don't forget demo roles)